lib/person.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+require 'date'
+require './lib/library.rb'
+
+def books
+  YAML.load_file('./lib/books.yml')
+end
+
+class Person
+  attr_accessor :name, :book
+
+  def initialize(name)
+    @name = name
+    @book = []
```

Comment:
I see a naming problem here. A Person could potentially have several books. I would name this in plural.